test/scripts/download-ethereum-client-test-files.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+# Always run the commands from the "test" dir
+cd $(dirname $0)/..
+
+# Grab Polkadot version
+branch=$(egrep -o '/polkadot-sdk.*#([^\"]*)' ../Cargo.lock | head -1)
+polkadot_release=$(echo $branch | sed 's/.*branch=//' | sed 's/#.*//')
+mkdir -p tmp
+wget -O - tmp/ethereum_client_test https://github.com/moondance-labs/polkadot-sdk/archive/$polkadot_release.tar.gz | tar -xz --strip=6 "polkadot-sdk-$polkadot_release/bridges/snowbridge/pallets/ethereum-client/tests/fixtures"
```

Comment:
The `tmp/ethereum_client_test` arg here is wrong? Running the script show this on my machine (but it works):

```
--2024-10-18 15:54:00--  http://tmp/ethereum_client_test
Resolviendo tmp (tmp)... falló: Fallo temporal en la resolución del nombre.
wget: no se pudo resolver la dirección del equipo ‘tmp’
--2024-10-18 15:54:00--  https://github.com/moondance-labs/polkadot-sdk/archive/tanssi-polkadot-stable2407.tar.gz
Resolviendo github.com (github.com)... 140.82.121.4
```

Also would be nice to avoid http requests if the files already exist, that probably needs storing the release somewhere to know when to update them.

Reply:
that is weird, I dont get that line :(

Reply:
Different versions of wget? :/

Reply:
fixed the not downloading part